sycl/doc/extensions/experimental/sycl_ext_oneapi_work_group_static.asciidoc

```diff
@@ -40,16 +40,10 @@ This extension is written against the SYCL 2020 revision 8 specification.  All
 references below to the "core SYCL specification" or to section numbers in the
 SYCL specification refer to that revision.
 
-The following extensions are required only for dynamic allocations:
-
-- link:../experimental/sycl_ext_oneapi_properties.asciidoc[sycl_ext_oneapi_properties]
-
-- link:../experimental/sycl_ext_oneapi_kernel_properties.asciidoc[sycl_ext_oneapi_kernel_properties]
-
 
 == Status
 
-This is a proposed extension specification, intended to gather community
+This is an experimental extension specification, intended to gather community
```

Comment:
Update to the correct text for an experimental extension.  (See my comment on the other spec in this PR.)